src/app/[lang]/(protected)/create-new-password/newPassPut.ts

```diff
@@ -0,0 +1,35 @@
+"use server";
+
+import { cookies } from "next/headers";
+import { redirect } from "next/navigation";
+import { env } from "@/env.mjs";
+import { getJwtAccessToken } from "../regenerateAccessToken";
+
+export async function newPassPut(
+  data: string,
+  lang: "en" | "uk",
+): Promise<string> {
+
+  const jwtAccessToken = await getJwtAccessToken();
+
+  if (!jwtAccessToken) {
+    redirect(`/${lang}`);
+  }
+
```

Comment:
You don't need to check if jwtAccessToken has a falsy value because this function returns only a string